list_of_projects_converter.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+file = open('list_of_projects.txt', 'w')
+
+filecut = []
+
+for files in os.listdir('./_projects'):
+    if files.endswith(".md"):
+        filecut.append(files)
+
+
+for sFile in filecut:
+    content = open('./_projects/'+ sFile, 'r')
+    filename = ''
+    membersfound = 0
+    members = ''
+    for line in content:
```

Comment:
Look for the second appearance of a line containing solely `---`: after that, leave the loop to avoid reading all of the project report. We are only interested in the header!